apache/docker-entrypoint.sh

```diff
@@ -16,9 +16,9 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
         group="$(id -g)"
     fi
 
-    chown www-data:www-data /sessions /var/nginx/client_body_temp
+    chown ${user}:${group} /sessions /var/nginx/client_body_temp
```

Comment:
@J0WI That would make sense, do you agree ?

Reply:
I think these directories are not longer used.

Reply:
Makes sense, will remove the line

Reply:
There are a few other references to them. The session can still be customized by mounting a specific config.

Reply:
Ah, I will research and try to add a comment